$NetBSD: patch-ad,v 1.3 2009/02/16 22:29:49 tnn Exp $

Prevent configure script from eating our default optimiser and CPU flags.

--- configure.orig	2009-02-13 17:43:45.000000000 +0100
+++ configure
@@ -21540,8 +21540,8 @@ _ACEOF
 esac
 
 if test "x$GCC" = "xyes" ; then
-    CFLAGS="-std=gnu99 -ggdb3 -Wall -Wextra -Wredundant-decls -Wpointer-arith -Wformat-security -W -Wmissing-declarations -Wdeclaration-after-statement -Wcast-align -Winit-self -Wundef -Wnested-externs -Wmissing-format-attribute"
-    CXXFLAGS="-Wall -Wformat-security -W"
+    CFLAGS="${CFLAGS} -std=gnu99 -ggdb3 -Wall -Wextra -Wredundant-decls -Wpointer-arith -Wformat-security -W -Wmissing-declarations -Wdeclaration-after-statement -Wcast-align -Winit-self -Wundef -Wnested-externs -Wmissing-format-attribute"
+    CXXFLAGS="${CXXFLAGS} -Wall -Wformat-security -W"
 fi
 
 { $as_echo "$as_me:$LINENO: checking for ANSI C header files" >&5
